substrate/frame/identity/src/lib.rs

```diff
@@ -150,6 +165,10 @@ pub mod pallet {
 		#[pallet::constant]
 		type ByteDeposit: Get<BalanceOf<Self>>;
 
+		/// The amount held on deposit per registered username.
```

Comment:
Please add a comment about when this is allowed to change: every block, in runtime upgrades or never.

Reply:
Done.